templates/todo/projects/java-mongo-aca/README.md

```diff
@@ -47,17 +47,25 @@ The following prerequisites are required to use this application. Please ensure
 The fastest way for you to get this application up and running on Azure is to use the `azd up` command. This single command will create and configure all necessary Azure resources - including access policies and roles for your account and service-to-service communication with Managed Identities.
 
 1. Open a terminal, create a new empty folder, and change into it.
-1. Run the following command to initialize the project, provision Azure resources, and deploy the application code.
+2. Run the following command to initialize the project.
 
 ```bash
-azd up --template todo-java-mongo-aca
+azd init --template todo-java-mongo-aca
 ```
 
-You will be prompted for the following information:
+This command will clone the code to your current folder and prompt you for the following information:
 
 - `Environment Name`: This will be used as a prefix for the resource group that will be created to hold all Azure resources. This name should be unique within your Azure subscription.
+
+3. Run the following command to package a deployable copy of your application, provision the template's infrastructure to Azure and also deploy the application code to those newly provisioned resources.
+
+```bash
+azd up
+```
+
+This command will prompt you for the following information:
+
 - `Azure Location`: The Azure location where your resources will be deployed.
-- `Azure Subscription`: The Azure Subscription where your resources will be deployed.
 
```

Comment:
It might be good to switch them in the order they are requested. Hence, Azure Subscription should be listed first

Reply:
@hemarina , I think there's a note for this template about the `location` and the note is right next to the location. That's why the Subscription is moved after the note.

I still think we should list the subscription first. But it's really a nit.

Reply:
@savannahostrowski , what if we create a headline for all Readme files to group `important notes` about the template.
We are currently adding important notes like this about the supported location which might be camouflaged.  It could be better to have an special and known section where users can see all notes and constrains for the template, all together. 
Of course, not for this PR, but that's the idea in case you want to put it in the queue.

Reply:
@vhvb1989 I think that's a good idea. Maybe we can file an issue and discuss/handle in a separate PR?